daemon/hostdaemon.go

```diff
@@ -51,21 +56,37 @@ func NewHostDaemon(vsp plugin.VendorPlugin) *HostDaemon {
 	}
 }
 
+func (d *HostDaemon) ensureConnected() {
+	if d.conn != nil {
+		return
+	}
+	retryPolicy := `{
+		"methodConfig": [{
+		  "waitForReady": false,
```

Comment:
Add a comment about waitForReady. It is not obvious what this does. We should say the effect is this makes Dial nonblocking and returns immediately.